python_ta/__init__.py

```diff
@@ -346,24 +355,25 @@ def get_file_paths(rel_path: AnyStr) -> Generator[AnyStr, None, None]:
                 yield os.path.join(root, filename)  # Format path, from root.
 
 
-def _verify_pre_check(filepath: AnyStr) -> bool:
+def _verify_pre_check(filepath: AnyStr, linter: PyLinter) -> bool:
```

Comment:
Rather than pass in the `linter` (which is a bit of overkill), instead pass in a `bool` for the actual value of the configuration option. Also, make sure to document this additional parameter in the function docstring.